src/esp/assets/GenericMeshData.cpp

```diff
@@ -74,5 +74,13 @@ void GltfMeshData::setMeshData(Magnum::Trade::AbstractImporter& importer,
     collisionMeshData_.indices = indexData_ = meshData_->indicesAsArray();
 }
 
+void GenericMeshData::setMeshData(Magnum::Trade::AbstractImporter& importer,
+                                  const std::string& meshName) {
+  // make sure name is appropriate for importer
+  int meshID = importer.meshForName(meshName);
+  CORRADE_ASSERT(meshID != -1, "Unknown meshName : " << meshName, );
```

Comment:
```suggestion
  CORRADE_ASSERT(meshID != -1, "Unknown meshName :" << meshName, );
```

(spaces are inserted automatically in between)